### DISCUSSION TITLE
(mini.ai) Inverse of composed patters

### DISCUSSION THREAD

**Discussion by IndianBoy42** [nvim-mini/mini.nvim#415] [Q&A] [answered]
Composed Patterns allow `(a || b) && (c && d) && ...`. How can I do `(a && b) || (c && d) || ...`?

**Comment by echasnovski** [comment, marked as answer]
Try wrapping in another `{}`? Like [here](https://github.com/echasnovski/mini.nvim/blob/2b37c5e1d2a3f62ba4060a819f17d8d976018928/doc/mini-ai.txt#L223-L227) but without `'.....'`. Without actual example it is hard to be more specific.

**Reply by IndianBoy42** [reply]
Oh that works, sorry I didn't look closely enough at the example (hard to tell without highlighting of the brackets) and just read the description that said cartesian product
